.github/workflows/run-unit-tests.yml

```diff
@@ -0,0 +1,18 @@
+name: Manually Run Unit Tests
+
+on:
+  push:  # FIXME: temporarily for dev, generally remove when done
```

Comment:
```suggestion
```

We don't need to run the unit tests on direct pushes to the repo if we're running them on PRs